apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/components/block-iframe-preview.js

```diff
@@ -206,6 +207,7 @@ const BlockFramePreview = ( {
 				title={ __( 'Preview', 'full-site-editing' ) }
 				className={ classnames( 'editor-styles-wrapper', className ) }
 				style={ style }
+				tabindex="-1"
```

Comment:
This line is the only relevant change, rest is just the build system updating the docs.